.nvmrc

```diff
@@ -0,0 +1 @@
+12.*
```

Comment:
could you add a newline at the end of this file please?

Reply:
done! i guess eslint/prettier didn't pick it up because it's at monorepo root, not sure if there is any config that would help with this